__docs__/wonder-blocks-tabs/tabs.stories.tsx

```diff
@@ -10,16 +11,78 @@ import Link from "@khanacademy/wonder-blocks-link";
 import {TextField} from "@khanacademy/wonder-blocks-form";
 import Tooltip from "@khanacademy/wonder-blocks-tooltip";
 import {Popover, PopoverContent} from "@khanacademy/wonder-blocks-popover";
+import {ScenariosLayout} from "../components/scenarios-layout";
+import {
+    longText,
+    longTextWithNoWordBreak,
+} from "../components/text-for-testing";
+import {IconMappings} from "../wonder-blocks-icon/phosphor-icon.argtypes";
+import {PhosphorIcon} from "@khanacademy/wonder-blocks-icon";
+import {PropsFor, View} from "@khanacademy/wonder-blocks-core";
+import {sizing} from "@khanacademy/wonder-blocks-tokens";
+import {Placeholder} from "../components/placeholder";
+
+const generateTabs = (
+    count: number,
+    tabContent: string = "Tab",
+    withIcons: boolean = false,
+) => {
+    return new Array(count).fill(0).map((_, index) => ({
+        label: (
+            <View
+                style={{
+                    gap: sizing.size_080,
+                    alignItems: "center",
+                    flexDirection: "row",
+                }}
+            >
+                {withIcons && <PhosphorIcon icon={IconMappings.cookie} />}
+                {`${tabContent} ${index + 1}`}
+                {withIcons && <PhosphorIcon icon={IconMappings.iceCream} />}
+            </View>
+        ),
+        id: `tab-${index + 1}`,
+        panel: <Placeholder>Tab contents {index + 1}</Placeholder>,
+    }));
+};
 
 const tabs: TabItem[] = [
-    {label: "Tab 1", id: "tab-1", panel: <div>Tab contents 1</div>},
-    {label: "Tab 2", id: "tab-2", panel: <div>Tab contents 2</div>},
-    {label: "Tab 3", id: "tab-3", panel: <div>Tab contents 3</div>},
+    {
+        label: "Tab 1",
+        id: "tab-1",
+        panel: <Placeholder>Tab contents 1</Placeholder>,
+    },
+    {
+        label: "Tab 2",
+        id: "tab-2",
+        panel: <Placeholder>Tab contents 2</Placeholder>,
+    },
+    {
+        label: "Tab 3",
+        id: "tab-3",
+        panel: <Placeholder>Tab contents 3</Placeholder>,
+    },
 ];
 
+function ControlledTabs(props: PropsFor<typeof Tabs>) {
+    const [selectedTabId, setSelectedTabId] = React.useState(
+        props.selectedTabId,
+    );
+
+    return (
+        <Tabs
+            {...props}
+            selectedTabId={selectedTabId}
+            onTabSelected={setSelectedTabId}
+            tabs={props.tabs}
+        />
+    );
+}
+
 export default {
-    title: "Packages / Tabs / Tabs / Tabs",
+    title: "Packages / Tabs / Tabs",
```

Comment:
yay 🎉

Reply:
I was able to remove one level because I created a "Subcomponents" level for the `Tab` component instead!